src/main/resources/avro/genotypephenotype.avdl

```diff
@@ -0,0 +1,176 @@
+@namespace("org.ga4gh.models")
+/**
+
+This protocol defines the associations between genotype
+and phenotype (G2P).  Associations can be made as a
+result of literature curation, computational modeling,
+inference, etc., and modeled and shared using this schema.
+
+Here, we follow the dogma of:
+
+      Genotype + Environment = Phenotype
+
+where a G2P association is between the G(enotype) in the context of
+some E(environment), which gives rise to a P(henotype). These
+associations have further evidence, provenance, and attribution.
+
+We leverage the GenomicFeature in the sequenceAnnotation schema here
+as it can accomodate any genomic feature from a single nucleotide variation
+(SNV), up through a gene, and/or complex rearrangements.  Each can
+be modeled as genomic features, and generally linked to a phenotype.
+Collections of these features can represent a genotype at different levels
+of completeness.  Therefore, we can represent single allelic variation,
+allelic complement, and multiple variants in a genotype that can each or
+collectively be associated with a phenotype.
+
+To enable standardized integration, this schema relies heavily on
+OntologyTerms, for typing phenotype, genomic features, and levels
+of evidence.  Suggested ontologies to leverage include (with browser links):
+Human Phenotype Ontology (HPO): http://www.ontobee.org/browser/index.php?o=hp
+Disease Ontology (DO): http://purl.obolibrary.org/obo/DOID_4
+Sequence Ontology (SO): http://www.sequenceontology.org/browser/
+Evidence Code Ontology (ECO): http://www.ontobee.org/browser/index.php?o=ECO
+Phenotypic Qualities (PATO): http://www.ontobee.org/browser/index.php?o=PATO
+*/
+protocol GenotypePhenotype {
+
+import idl "common.avdl";
+import idl "metadata.avdl";
+import idl "sequenceAnnotations.avdl";
+
+
+/**
+The context in which a genotype gives rise to a phenotype.
+This is fairly open-ended; as a stub we have a simple ontology term.
+For example, a controlled term for a drug, or perhaps an instance of a 
+complex environment including temperature and air quality, or perhaps
+the anatomical environment (gut vs tissue type vs whole organism).
+*/
+record EnvironmentalContext {
+
+  /** The Environment ID. */
+  union { null, string } id = null;
+
+  /**
+  Examples of some environment types could be drawn from:
+  Ontology for Biomedical Investigations (OBI): http://purl.obofoundry.org/obo/obi/browse
+  Chemical Entities of Interest (ChEBI): http://www.ontobee.org/browser/index.php?o=chebi
+  Environment Ontology (ENVO):  http://www.ontobee.org/browser/index.php?o=ENVO
+  Anatomy (Uberon): http://www.ontobee.org/browser/index.php?o=uberon
+  */
+  org.ga4gh.models.OntologyTerm environmentType;
```

Comment:
I think you're already in the `org.ga4gh.models` namespace, so this doesn't need to be fully qualified.